tests/sentry/incidents/test_logic.py

```diff
@@ -656,6 +661,110 @@ def test_on_demand_metric_alert(self, mocked_schedule_update_project_config):
         mocked_schedule_update_project_config.assert_called_once_with(alert_rule, [self.project])
 
 
+class TestSendIncidentAlertNotification(TestCase):
+    @cached_property
+    def alert_rule(self):
+        return self.create_alert_rule()
+
+    @cached_property
+    def discord_info(self):
+        return {
+            "channel_id": "channel-id",
+            "guild_id": "example-discord-server",
+            "guild_name": "Server Name",
+        }
+
+    @cached_property
+    def integration(self):
+        return Integration.objects.create(
+            provider="discord",
+            name="Example Discord",
+            external_id=self.discord_info["guild_id"],
+            metadata={
+                "guild_id": self.discord_info["guild_id"],
+                "name": self.discord_info["guild_name"],
+            },
+        )
+
+    @cached_property
+    def action(self):
+        type = AlertRuleTriggerAction.Type.DISCORD
+        target_type = AlertRuleTriggerAction.TargetType.SPECIFIC
+        return create_alert_rule_trigger_action(
+            self.trigger,
+            type,
+            target_type,
+            target_identifier=self.discord_info["channel_id"],
+            integration_id=self.integration.id,
+        )
+
+    @cached_property
+    def trigger(self):
+        return create_alert_rule_trigger(self.alert_rule, "hello", 1000)
+
+    def test_integration_inactive(self):
```

Comment:
How does this make the integration inactive?

Reply:
I think the wording and intention here is off. I just wanted to test for the case `if not channel` but that would be better tested by just setting the action target identifier to null. This case may be unnecessary if the `if not channel` check is removed.

Reply:
Ah ok - if there's ambiguity behind the naming I usually add a docstring to explain

Reply:
I actually think this test is not needed. The `send_incident_alert_notification` functions for the other integrations do  not specifically test this method since it is a helper.